index.html

```diff
@@ -4,10 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/styles.css">
   <title>Luis Zubia First Porfolio</title>
 </head>
 <body>
 
+  <header class="header">
+    <p>Luis Zubia</p>
+    <nav>
+      <a href="#">
+        <img class="icon" src="/imgs/menu-icon.png" alt="Menu icon">
+      </a>
+    </nav>
+  </header>
+
+  <section class="headline">
+    <h1>Hey There. I'm Luis</h1>
+    <h2>I’am a Software Developer</h2>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <ul>
+      <li>
+        <a href="#">
+          <img class="icon" src="/imgs/github_icon.svg" alt="GitHub icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="icon" src="/imgs/linkedin_icon.svg" alt="LinkedIn icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="icon" src="/imgs/angelist_icon.svg" alt="Angelist icon">
+        </a>
+      </li>
+      <li>
+        <a href="#">
+          <img class="icon" src="/imgs/twitter_icon.svg" alt="Twitter icon">
+        </a>
```

Comment:
- [ ] You did a great job however, you didn't pay attention to a small mistake in the 4 social media icons so the icons are not displaying. In order to fix that issue just remove `/` from the image link, `src="imgs/linkedin_icon.svg"` instead of src="/imgs/linkedin_icon.svg".